app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -67,6 +77,25 @@ class UploadBddInstructions < Job
         }
       )
 
+      # # Returns the correct SupplementalDocumentUploadProvider based on the state of the
+      # # ApiProviderFactory::FEATURE_TOGGLE_UPLOAD_BDD_INSTRUCTIONS feature flag for the current user
+      # #
+      # @return [EVSSSupplementalDocumentUploadProvider or LighthouseSupplementalDocumentUploadProvider]
+      def self.api_upload_provider(submission)
+        user = User.find(submission.user_uuid)
+
+        ApiProviderFactory.call(
+          type: ApiProviderFactory::FACTORIES[:supplemental_document_upload],
+          options: {
+            form526_submission: submission,
+            va_document_type: BDD_INSTRUCTIONS_DOCUMENT_TYPE,
+            statsd_metric_prefix: STATSD_KEY_PREFIX
```

Comment:
Note passing more initializer options to the provider now; we need this data for logging and, in the Lighthouse case, polling

Reply:
Why do we need to pass in statsd prefix? Is the statsd prefix not just hard-coded into each provider? 
IE evss provider has a evss prefix constant, lh has a lh prefix constant?